src/backend/aspen/api/utils/authz.py

```diff
@@ -39,56 +39,6 @@ def authz_sample_filters(query: Query, sample_ids: Set[str], user: User) -> Quer
     return query
 
 
-def authz_samples_cansee(
```

Comment:
We moved this functionality to a different util module since it doesn't need authz logic anymore